### PR TITLE
[Feature](topn) change topn_filter_limit_threshold to topn_filter_ratio

### PR DESCRIPTION
**Pull request by BiteTheDDDDt**
## Proposed changes
change topn_filter_limit_threshold to topn_filter_ratio

### CODE REVIEW

**Comment by BiteTheDDDDt** [comment]
run buildall

**Comment by BiteTheDDDDt** [comment]
run buildall

**Comment by BiteTheDDDDt** [comment]
run buildall

File: fe/fe-core/src/main/java/org/apache/doris/qe/SessionVariable.java [hunk 1]
```diff
@@ -1520,8 +1520,8 @@ public void setEnableLeftZigZag(boolean enableLeftZigZag) {
     public boolean enableTwoPhaseReadOpt = true;
     @VariableMgr.VarAttr(name = TOPN_OPT_LIMIT_THRESHOLD)
     public long topnOptLimitThreshold = 1024;
-    @VariableMgr.VarAttr(name = TOPN_FILTER_LIMIT_THRESHOLD)
-    public long topnFilterLimitThreshold = 10240000;
```
[the remark this answers is not in the document]
Comment (BiteTheDDDDt) [reply fe/fe-core/src/main/java/org/apache/doris/qe/SessionVariable.java:1524]: Now this parameter is divided into topnFilterLimitThreshold and topnOptLimitThreshold. topnFilterLimitThreshold only affects topn filter planning.

**Review by englefly — APPROVED** [review]

**Review by zclllyybb — APPROVED** [review]
LGTM

**Pull request merged**